classes/services/queryBuilders/PKPStatsGeoQueryBuilder.php

```diff
@@ -178,14 +183,17 @@ protected function _getObject(): Builder
             if (Application::get()->getName() == 'omp') {
                 $sectionColumn = 'p.series_id';
```

Comment:
I recommend turning `$sectionColumn` into a property of `APP\services\queryBuilders\StatsGeoQueryBuilder`.